package.json

```diff
@@ -84,6 +84,6 @@
     "sinon": "^1.17.5"
   },
   "peerDependencies": {
-    "react": "0.13.x || 0.14.x || ^15.0.0-0"
+    "react": "0.13.x || 0.14.x || ^15.x"
```

Comment:
No it's not anymore. I did it because I wrote that line before the release of 15.2. I was working with 15.2.0-rc.1
 or 2, which doesn't match ^15.0.0-0). I added a commit to fix that.